nltk/corpus/__init__.py

```diff
@@ -54,8 +54,8 @@
 ``nltk.corpus.brown.words()``:
 
     >>> from nltk.corpus import brown
-    >>> print(", ".join(brown.words()))
-    The, Fulton, County, Grand, Jury, said, ...
+    >>> print(", ".join(brown.words()[:6])) # only first 6 words
+    The, Fulton, County, Grand, Jury, said
```

Comment:
It was my understanding that `...` is valid for tests, as long as you use `# doctest: +ELLIPSIS`. That seems cleanest, especially for cases where the alternative is really long.

Reply:
Okay, I agree that's the easier and cleaner solution. How do I update this? Do I have to create a new pull request?

Reply:
No need! You can push new commits to the `docstring-tests` branch on your nltk fork, and they will automatically be included in this PR.